src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -81,6 +81,12 @@
     <ProjectCapability Include="ProjectImportsTree" />
     <ProjectCapability Include="LaunchProfiles" />
     <ProjectCapability Include="NoGeneralDependentFileIcon"/>
+    <!--
+      We want the tree to respect IProjectTree2.DisplayOrder when it is set.
+      This is different to capability EditableDisplayOrder, used by F# for example, which
+      allows the user to manipulate display order values.
+    -->
+    <ProjectCapability Include="SortByDisplayOrder"/>
```

Comment:
F# has this as a fixed capability (comes from project type) probably for a reason, it probably cannot be dynamic (comes from targets).

Reply:
Have made this a fixed capability.